safe-ledger/cli/sign_with_cast_wallet.py

```diff
@@ -0,0 +1,64 @@
+import json
+import subprocess
+import sys
+import typing
+
+
+def sign_safe_tx(safe_tx_data: dict[str, typing.Any],
+                 account_name: str) -> str | None:
+    safe_tx_hex = json.dumps(safe_tx_data)
+    cmd = [
```

Comment:
once `--ledger` supports clear signing, we could switch to this approach to use ledger too:)

Reply:
Yep, that would be nice